financial-connections/src/main/java/com/stripe/android/financialconnections/features/institutionpicker/InstitutionPickerViewModel.kt

```diff
@@ -136,6 +110,49 @@ internal class InstitutionPickerViewModel @Inject constructor(
         )
     }
 
+    private fun determineBackNavigation() {
+        suspend {
+            val manifest = getOrFetchSync().manifest
+            val isLinkConsumer = manifest.accountholderIsLinkConsumer ?: false
+            val isNetworkingFlow = manifest.isNetworkingUserFlow ?: false
+            isLinkConsumer && !isNetworkingFlow
+        }.execute {
+            val disableBackNavigation = it() ?: true
```

Comment:
I don't think it can happen, but in case of an error we won't have logs (and the back button will be disabled).

An alternative could be to move this inside the payload and keep a unique load function, then read the boolean as `payload()?.disableBackNavigation ?: true`.